WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -1243,11 +1243,19 @@ protected void showInvalidUsernameOrPasswordDialog() {
         bundle.putString(SignInDialogFragment.ARG_OPEN_URL_PARAM, getForgotPasswordURL());
         bundle.putString(ENTERED_URL_KEY, EditTextUtils.getText(mUrlEditText));
         bundle.putString(ENTERED_USERNAME_KEY, EditTextUtils.getText(mUsernameEditText));
+        passHelpshiftErrorOriginTag(bundle);
         nuxAlert.setArguments(bundle);
         ft.add(nuxAlert, "alert");
         ft.commitAllowingStateLoss();
     }
 
+    protected void passHelpshiftErrorOriginTag(Bundle bundle) {
+        bundle.putSerializable(HelpshiftHelper.ORIGIN_KEY,
+                isWPComLogin() ? Tag.ORIGIN_LOGIN_SCREEN_ERROR_WPCOM
+                        : (isJetpackAuth() ? Tag.ORIGIN_LOGIN_SCREEN_ERROR_JETPACK
```

Comment:
`isJetpackAuth()` will be true only if the user comes from the Stats screen (he's requested to sign in to his wpcom account to see his jetpack site stats), and this is pretty rare.

I think it's OK since it's used in Helpshift only (and so to give more context to HE), but this should not be used for stats. Adding a comment on top of that code would be great.